### DISCUSSION TITLE
Images in horizontal align

### DISCUSSION THREAD

**Discussion by JGamonalL** [awcodes/filament-tiptap-editor#352] [General]
Hi there! What is the best way to leave the images horizontally aligned when using the editor? I tried using the grid but it doesn't work and the code of the images is displayed in horizontal order

**Comment by awcodes** [comment]
What do you mean "grid isn't working"? Can you share some screenshots?

**Reply by JGamonalL** [reply]
Thanks for your response!! I refer to the Grid Builder. Here are some screenshots of my attempts to make a horizontal line of pictures.

This is the text inside Tip Tap editor
![image](https://github.com/awcodes/filament-tiptap-editor/assets/162324589/bec2793f-701a-4d91-82e2-76bb13f78e64)

And this is the output
![image](https://github.com/awcodes/filament-tiptap-editor/assets/162324589/dfaaa78d-e531-4e0f-9d01-a9dbdf46652a)

**Reply by awcodes** [reply]
Ah, yea, you are responsible for styling the output appropriately for your needs. The plugin can presume to know anything about the display of the html content outside of the editor itself.

**Reply by awcodes** [reply]
you could reference the plugin's stylesheet if you need a place to start with building your own css for it.

**Reply by JGamonalL** [reply]
sorry i didn't reply before. Thanks for your answers. I did use some custom styling and that gave me the desired output. Thank you very much!
